### PR TITLE
[13.0][MIG] sale_timesheet_order_line_sync: Migration to 13.0

### PR DESCRIPTION
**Pull request by joao-p-marques**
TT25578

ping @carlosdauden @rafaelbn

### CODE REVIEW

File: sale_timesheet_order_line_sync/tests/test_sale_timesheet_order_line_sync.py [hunk 1]
```diff
@@ -121,14 +123,14 @@ def test_sale_timesheet_order_line_sync(self):
...
                 "partner_id": partner.id,
                 "partner_invoice_id": partner.id,
                 "partner_shipping_id": partner.id,
             }
         )
-        sale_order_line2 = self.SudoSaleOrderLine.create(
+        sale_order_line2 = self.sudo_sale_order_line.create(
             {
                 "order_id": sale_order.id,
                 "name": product.name,
```
Comment (victoralmau) [inline sale_timesheet_order_line_sync/tests/test_sale_timesheet_order_line_sync.py:134]: IMO although not appear in Guidelines, uppercase in variables related to models are allowed ant all these changes aren't necessary.

Comment (pedrobaeza) [reply sale_timesheet_order_line_sync/tests/test_sale_timesheet_order_line_sync.py:134]: Yes, please reduce the diff at maximum

Comment (joao-p-marques) [reply sale_timesheet_order_line_sync/tests/test_sale_timesheet_order_line_sync.py:134]: Ok, thanks for the suggestions. Changes made-

**Review by victoralmau — APPROVED** [review]

**Review by pedrobaeza — APPROVED** [review]

**Pull request merged**